src/ifp/test/init_floorplan1.py

```diff
@@ -0,0 +1,27 @@
+from openroad import *
```

Comment:
do we want to use pep-8 here? https://peps.python.org/pep-0008/

Reply:
Done

Reply:
sorry I didn't meant just import, but also the class/method/variable naming scheme :)

Reply:
In c++ the standard naming (from google style) is incompatible with pep8 naming.  We would have to rename every method in swig which seems unpleasant.  Do you have a better suggestion?